website/content/docs/commands/events.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: events - Command
+description: |-
+  The "events" command interacts with the Vault events subsystem.
+---
+
+# events
+
+The `events` command is used to subscribe to Vault
+[events](/vault/docs/concepts/events) via `events subscribe`. It shows a
+real-time display of events generated by Vault on standard out.
+
+The desired event types (sometimes called "topics") must be specified, which
+may be a glob pattern (with `*` treated as a wildcard).
+
+The output will be JSON objects serialized using the default protobuf JSON
+serialization format, with one line per event received.
+
+Note that this command is designed to run indefinitely. It is similar to
+`tail -f` in the Unix world. This command will not exit on its own unless it
+encounters an unexpected error. As a user, you must terminate this process
+yourself to shut it down.
```

Comment:
```suggestion
```

Suggest including the behavioral note with the intro sentence so folks have a clear idea of how the command behaves right from the jump.